src/fontra/views/editor/scene-controller.js

```diff
@@ -1333,6 +1371,70 @@ function getSelectedContours(path, pointSelection) {
   return [...selectedContours];
 }
 
+function getSelectedClosableContours(path, pointSelection) {
+  if (!path || !pointSelection) {
+    return [];
+  }
+  const selectedContours = new Set();
+  for (const contourIndex of getSelectedContours(path, pointSelection)) {
+    if (path.contourInfo[contourIndex].isClosed) {
+      // skip if contour is closed already
+      continue;
+    }
+    if (path.getNumPointsOfContour(contourIndex) <= 2) {
+      // skip if contour has two (or less) points only
+      // (two on-curve or one off-curve and one on-curve)
+      continue;
+    }
+    const contour = path.getContour(contourIndex);
+    const numOnCurvePoints = contour.pointTypes.reduce(
+      (acc, pointType) =>
+        acc +
+        ((pointType & VarPackedPath.POINT_TYPE_MASK) === VarPackedPath.ON_CURVE
+          ? 1
+          : 0),
+      0
+    );
+    if (numOnCurvePoints === 1) {
```

Comment:
Better safe than sorry:
```suggestion
    if (numOnCurvePoints <= 1) {
```